.github/workflows/release.yml

```diff
@@ -31,7 +31,10 @@ jobs:
       - name: Package
         run: |
           mvn clean package -DskipTests
-          mv "$(find hugegraph-*.tar.gz | head -1)" hugegraph-${TAG_NAME/#v/}.tar.gz
+
+      - name: Rename file
+        if: contains(env.TAG_NAME, "-")
```

Comment:
only need to rename when TAG_NAME is like 0.12.0-b2?

Reply:
yes